examples/nanoChatGPT/env.py

```diff
@@ -46,20 +48,15 @@ def _step(self, tensordict):
 @torch.no_grad()
 def _reset(self, tensordict):
     self.step_num = 0
-    if tensordict is None or tensordict.is_empty():
-        # if no tensordict is passed, we generate a single set of hyperparameters
-        # Otherwise, we assume that the input tensordict contains all the relevant
-        # parameters to get started.
-        tensordict = TensorDict({}, batch_size=self.config["batch_size"])
-
     batch = next(self.dataloader)
 
+    prompt = batch.prompt[:self.batch_size[0], -self.block_size :]
```

Comment:
why we need this? `:self.batch_size[0]`

Reply:
we can drop it
It was because I was using the same dataloader for the test env and the other, but now I have changed the config of the test so it's not needed anymore